django/utils/crypto.py

```diff
@@ -74,3 +74,22 @@ def pbkdf2(password, salt, iterations, dklen=0, digest=None):
     password = force_bytes(password)
     salt = force_bytes(salt)
     return hashlib.pbkdf2_hmac(digest().name, password, salt, iterations, dklen)
+
+
+try:
+    _ = hashlib.md5(usedforsecurity=False)  # nosec
+
+    def md5(string=b'', usedforsecurity=True):
+        """Return an md5 hashlib object using usedforsecurity parameter
+        For python distributions that support the usedforsecurity keyword
+        parameter, this passes the parameter through as expected.
+        See https://bugs.python.org/issue9216
+        """
+        return hashlib.md5(string, usedforsecurity=usedforsecurity)  # nosec
+except TypeError:
+    def md5(string=b'', usedforsecurity=True):
```

Comment:
Since the first argument is a bytes-like, I would something other than "string." The `hashlib` docs use `data`, for example (see e.g. [here](https://docs.python.org/3/library/hashlib.html#hashlib.new)). Alternatively, it might be better just to pass along the `*args` and `**kwargs` to the function being wrapped (except for `usedforsecurity`).

Reply:
Done